ibft/commit.go

```diff
@@ -53,6 +53,9 @@ func (i *Instance) forceDecidedPipeline() pipeline.Pipeline {
 
 // CommittedAggregatedMsg returns a signed message for the state's committed value with the max known signatures
 func (i *Instance) CommittedAggregatedMsg() (*proto.SignedMessage, error) {
+	if i.State.DecidedMsg != nil {
+		return i.State.DecidedMsg, nil
+	}
 	if i.State.PreparedValue == nil {
```

Comment:
@nivBlox why do we need anything after i.State.DecidedMsg? if it's not decided than i.State.DecidedMsg is nil and so does the code after it

Reply:
done